src/VersionTool/VersionTool.csproj

```diff
@@ -3,15 +3,15 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
     <AssemblyName>VersionTool</AssemblyName>
     <OutputType>exe</OutputType>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.Cli.Utils" Version="1.0.0-preview2-1-003177" />
```

Comment:
The reason this was pinned to an old version of the package is that it is a project.json tool. I'm not sure this will work if you lift it to the 1.0.1 version of this package. It may be better to leave this one untouched until we completely fix it to work with csproj. See https://github.com/aspnet/BuildTools/issues/270 

cc @JunTaoLuo